ontology/spdx-ontology.owl.xml

```diff
@@ -380,8 +389,10 @@ Known issues:</rdfs:comment>
                 </owl:unionOf>
             </owl:Class>
         </rdfs:range>
-        <rdfs:comment xml:lang="en">Licensing information that was discovered directly in the subject file. This is also considered a declared license for the file.</rdfs:comment>
-        <ns:term_status>deprecated</ns:term_status>
+        <rdfs:comment xml:lang="en">Licensing information that was discovered directly in the subject file. This is also considered a declared license for the file.
+
+If the License Information in File field is not present for a file, it implies an equivalent meaning to NOASSERTION.</rdfs:comment>
```

Comment:
Comments refer to the RDF terms, so use `licenseInfoInFile` instead of `License Information in File`